.github/workflows/test.yml

```diff
@@ -0,0 +1,37 @@
+name: Tests
+
+on:
+  push:
+  schedule:
+    - cron: '0 0 * * *'
```

Comment:
This workflow will run whenever someone pushes a change, along with every day at midnight on the master branch.

This will update the badge in the readme if it's failing :)